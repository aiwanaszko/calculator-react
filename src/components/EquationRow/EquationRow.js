import React from "react";

export default function EquationRow(props) {
  return (
    <div>
      <input type="text" readOnly />
    </div>
  );
}
