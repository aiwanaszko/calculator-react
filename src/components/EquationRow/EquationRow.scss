.equation-row input {
  font-family: "Montserrat";
  width: 98%;
  text-align: right;
  background-color: black;
  border: none;
  height: 50px;
  color: grey;
  font-size: 24px;
  outline: none;
  border-radius: 5px;
}
