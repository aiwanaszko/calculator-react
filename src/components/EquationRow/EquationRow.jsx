import React from "react";
import "./EquationRow.scss";

export default function EquationRow(props) {
  return (
    <div className="equation-row">
      <input type="text" readOnly value={props.value} />
    </div>
  );
}
