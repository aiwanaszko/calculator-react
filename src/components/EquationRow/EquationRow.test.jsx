import React from "react";
import renderer from "react-test-renderer";
import EquationRow from "./EquationRow";

it("renders the EquationRow correctly", () => {
  const component = renderer.create(<EquationRow />).toJSON();
  expect(component).toMatchSnapshot();
});
