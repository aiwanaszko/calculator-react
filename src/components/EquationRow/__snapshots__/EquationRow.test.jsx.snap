// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the EquationRow correctly 1`] = `
<div
  className="equation-row"
>
  <input
    readOnly={true}
    type="text"
  />
</div>
`;
